.circleci/config.yml

```diff
@@ -193,38 +203,38 @@ jobs:
             retry sh -c 'echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin'
             if [ -n "$CIRCLE_TAG" ]; then
```

Comment:
Now we could actually refactor this, so that we set the correct `DOCKER_TAG` in the beginning (`CIRCLE_TAG` if it exists). Then we could simplify this part to:
```
retry docker-compose push webknossos
retry docker-compose push webknossos-datastore
retry docker-compose push webknossos-tracingstore
if [ -n "$CIRCLE_BRANCH" ]; then
  # tag and push wk, wk-ds, wk-ts, wk-dev with {NORMALIZED_BRANCH}
fi
```
@normanrz what do you think?

Reply:
like this?

Reply:
Awesome :heart: